plugin/cli/main.py

```diff
@@ -48,35 +48,12 @@ def render_new_file(renderer, response, resource):
   f.content = renderer.render(resource)
 
 
-def get_oneof_for_resource(collection_config, gapic_config):
-  oneof = None
-  for oneof_config in gapic_config.collection_oneofs.values():
-    for collection_name in oneof_config.collection_names:
-      if collection_name == collection_config.entity_name:
-        if oneof:
-          raise ValueError("A collection cannot be part of multiple oneofs")
-        oneof = oneof_config
-  return oneof
-
 def generate_resource_name_types(response, gapic_config, java_package):
-  renderer = pystache.Renderer(search_dirs=TEMPLATE_LOCATION)
-  for collection_config in gapic_config.collection_configs.values():
```

Comment:
This method has been refactored to gapic_utils.collect_resource_name_types(gapic_config, java_package), which just collects the resources.